deployment/clouddeploy/gke-workers/environments/oss-vdb-test/nvd-mirror.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1
```

Comment:
Also need to add `nvd-mirror.yaml` to the `kustomization.yaml` patches in `environments/oss-vdb-test/`

Reply:
Ugh, I always miss that one. Thanks! Fixed in [0215350](https://github.com/google/osv.dev/pull/1839/commits/0215350534c233f743f5332e10486892ee32e3fa)